uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-report_test.js

```diff
@@ -561,8 +561,8 @@
                 }));
               });
 
-              it('groups and flattens all breakdown entries before caching them', function () {
-                expect(controller.sections.expired.data.length).toBe(expectedFormat.length);
+              it('groups and flattens all breakdown and expired TOIL entries before caching them', function () {
+                expect(controller.sections.expired.data.length).toBe(expectedFormat.length + leaveRequestMock.all().values.length);
```

Comment:
We dont, but in this it wont make any difference as the `LeaveRequest.all` is mocked to return all leave requests.
Added unit test to check whether the API is getting called with `expired` parameters